### ISSUE TITLE
Question about database

### ISSUE THREAD

**Issue by paschoal-artur** [julius2503/higher-lower-game#2]
Hey man, I want to make a higher or lower game and I wanted to use your database, just need to know if you let me and from where is this data taken from, like can I use it to make my game or will some answers not make sense?

**Comment by julius2503** [comment]
Hey,
the data was collected from the bot itself. When I built it a couple of years ago it was the complete data-set, as it auto-played the game several times and did not find any new words. Unfortunately, I can’t tell you if the higher lower games team did add new words after all this years, as I haven’t used the bot again and it is probably not working anymore. But of course I am fine with letting you use the data yourself 😄

**Comment by paschoal-artur** [comment]
Thanks for you response mate!!
So the bot generates the values by himself while playing the game or does he get them from the game?
Really nice for it to be the complete data-set although it might be a little out of date, it's still awesome.

**Comment by julius2503** [comment]
Yes, it’s more or less brute force. It plays the game by itself and chooses with a little bit of stochastic if the word is higher or lower and adds it to the database. If it fails it just starts again. After some time, at best when run parallel, it found all words and automatically plays the game until the game breaks and you can’t choose anymore, that will happen at score 1556, and the lower/higher buttons disappears. 

here is a video where the program plays the game with the full database:  
https://youtu.be/1wf2AVbQTxY?si=OhcxW_KsYXPkJmMu

**Comment by paschoal-artur** [comment]
Real nice man, loved the application. So basically it's going to play the game until it has all the answers and values, right? 
Just another question, I want to make my data updated, so do you think I can try to run your bot, fix errors if there are any, and collect data from the game that's more up to date? 
Will be seeing your video, thanks a lot for your response!!
